### DISCUSSION TITLE
Welcome to vue-easytable Discussions!

### DISCUSSION THREAD

**Discussion by huangshuwei** [Happy-Coding-Clans/vue-easytable#350] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## 👋  欢迎！
  我们正在使用“讨论”作为与社区其他成员联系的地方。 在这里你可以：
  * 提出你在使用 vue-easytable 过程中遇到的问题。
  * 分享你对 vue-easytable 的反馈和建议。
  * 与社区的其他成员互动。

**Comment by ElectronSz** [comment]
Keep up with the good work

**Comment by jaywilburn** [comment]
Is there any plan to support Nuxt 3?
